FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/RequestedAttributes.java

```diff
@@ -23,6 +23,7 @@
     private final String relyingPartyUserId;
     private final String emailAddress;
     private final String organisationIdIdentifier;
+    private final OrganisationIdInfo organisationIdInfo;
```

Comment:
This one we should name organisationId, same way we used for DocumentInfo and attribute was document only.